persistence/azure-table/configuration_code_astp_[3,).partial.md

```diff
@@ -67,23 +67,21 @@ In case installers are enabled, but there's a need to opt out from creating the
 
 snippet: EnableInstallersConfigurationOptingOutFromTableCreation
 
-### Partitioning and compatibility helpers
+### Partitioning and compatibility mode helpers
 
-To atomically store data within the same transaction, all data written as part of the same message handling pipeline needs to share the same partition key.
-When storing sagas, saga IDs are by default deterministically derived from the saga data: the correlation property name and the correlation property value. Therefore, in conversations concerning a saga as part of the message handling pipeline, this deterministically derived ID is a perfect fit to be used as a partition key. Unfortunately, the saga ID assignment happens quite late during the message handling process and is not exposed to the user code.
+During a given message handling pipeline, multiple data storage operations may occur. To commit them atomically, in a single transaction, they must share a partition key. In conversations involving a saga, the saga ID is a good candidate for a partition key. Unfortunately, saga IDs are determined late in the message handling process and are not exposed to user code.
 
 This makes it difficult to:
 
-* Support transactional processing for existing sagas that were stored with the previous version of the persister and may have in-flight messages
-* Expose the deterministic saga ID early enough to enable business operations to participate in the transaction before the saga has been created
```

Comment:
> business operations

I believe it's better to avoid the term "business operations" as something distinct from "saga data" operations. Saga data may have just as much, if not more, "business" meaning than "other" data.

Reply:
We used the term "business data" also in the blog post quite intensively

https://particular.net/blog/transactions-with-azure-table-persistence

which passed through editorial review. Do we have any consistency or something we could refer to see what we use in other docs?

Reply:
@danielmarbach regardless of what's used elsewhere, does the use of the term add any value to (the new version of) the bullet point? I.e.

> * Allow other data operations to participate in the saga data transaction.

vs

* Allow business operations to participate in the saga data transaction.

I find "other data operations" more precise, and less open to questions like: is X a business operation or not? Ultimately, it doesn't matter. The important point seems to be that other data operations, regardless of what they are, can participate in the saga data transaction.

Reply:
Good arguments @adamralph. I'm 👍 for making the change